src/Bynder/Api/Impl/AbstractRequestHandler.php

```diff
@@ -1,26 +1,33 @@
 <?php
+
 namespace Bynder\Api\Impl;
 
 abstract class AbstractRequestHandler
 {
     protected $configuration;
 
-    public function sendRequestAsync($requestMethod, $uri, $options = [])
+    public function formRequest($uri, $requestMethod)
     {
-        $uri = sprintf(
-            'https://%s/%s', $this->configuration->getBynderDomain(), $uri
-        );
-
         if (!in_array($requestMethod, ['GET', 'POST', 'DELETE'])) {
             throw new Exception('Invalid request method provided');
         }
 
-        $request = $this->sendAuthenticatedRequest($requestMethod, $uri, $options);
+        return sprintf(
+            'https://%s/%s',
+            $this->configuration->getBynderDomain(),
+            $uri
+        );
+    }
+
+    public function sendRequestAsync($requestMethod, $uri, $options = [])
+    {
+        $composedUri = $this->formRequest($uri, $requestMethod);
+        $request = $this->sendAuthenticatedRequest($requestMethod, $composedUri, $options);
```

Comment:
```suggestion
        $request = $this->sendRequestAsyncRawResponse($requestMethod, $uri, $options);
```

Reply:
I will prune this as I wanted to use it to get the header for the correlation-id (I had used the `getHeader() `method instead). But with your comments from the python-sdk I decided to remove the returning of fileId and correlationId so this won't be required at all.